lang/python311/Portfile

```diff
@@ -2,11 +2,13 @@
 
 PortSystem 1.0
 PortGroup select 1.0
+PortGroup clang_dependency 1.0
 
 name                python311
 
 # Remember to keep py311-tkinter and py311-gdbm's versions sync'd with this
 version             3.11.6
+revision            1
```

Comment:
The revision should not be increased when only changing the compiler selection order.